### ISSUE TITLE
More customisation options for password generator

### ISSUE THREAD

**Issue by ankrman** [strongbox-password-safe/Strongbox#515]
Hey Mark,

I have to enter passwords on machines on which I don't have access to my database (so no auto-type) and which have foreign keyboard layouts. So I use the "magnifying" function to type them manually. 

Because of the different keyboard layouts it would be of great help, if I could create "compatible" passwords which only contain special characters that can be easier entered on most layouts I have to work with (e. g. ! and %). 

Long story short: an extra, customizable "also-use-this-chars" group for the PW generator like in KeePass (or inverted "exclude-this-chars" or even both) would be great. 

Thanks and keep up the great work!

**Comment by strongbox-mark** [comment]
Noted, thanks @ankrman

**Comment by ankrman** [comment]
Hey Mark,

as I'm still stuck to another PW generator, I'm hoping for my enhancement with every update. If the effort is too big, would it at least be possible to add a simple "unwanted characters" textbox to the PW generator?

Would be highly appreciated 🙏🏼

**Comment by strongbox-mark** [comment]
Hi @ankrman - Will try bump an "Excluded Characters List" up the priority list...

**Comment by ankrman** [comment]
Still hoping thus will ever come. As a reference, Keepassium has a nice implementation:
![image](https://user-images.githubusercontent.com/46501937/197381623-504b9998-f7b4-4098-864a-766709c64eb4.jpeg)

**Comment by ankrman** [comment]
Thanks for implementing the exclusion list ❤️
